### ISSUE TITLE
Global CLI? 

### ISSUE THREAD

**Issue by joshsmith01** [tonik/cli#12]
Should I be able to install this CLI globaly? i.e, at `~/.composer`? 

I assumed I would be able to so I did, but I can't use `tonik` to add new starter themes to my `~/sites/wordpress/wp-content/themes/` directory. 

I think it's a CLI so I should be able to install it where all my other CLIs are located. Why would I clone the starter theme and have the CLI (that builds the starter theme) in my starter theme. 

I'm so confused right now 😭

**Comment by jedrzejchalubek** [comment]
Good point. Indeed, we can do better here - labeled as the enhancement.

Unfortunately, this will probably introduce some breaking changes so it has to go to next major release :(
